.github/workflows/build.yml

```diff
@@ -48,7 +48,8 @@ jobs:
           sudo apt install -y \
             freeglut3-dev \
             libpng-dev \
-            libsqlite3-0 \
+            libsqlite3-dev \
```

Comment:
At some point you may want to add another job to the matrix, where these two don't get installed (to test the bundled sqlite wrap and code is working). Or perhaps instead, a separate weekly/biweekly cron job would be more appropriate since it's not likely to break very often.

If you agree, please open a ticket for your reference. :) I might do it. :)